ADAL/tests/integration/ios/ADAcquireTokenPkeyAuthTests.m

```diff
@@ -154,4 +112,43 @@ - (void)testTokenEndpointPkeyAuthNoWPJ
     XCTAssertEqualObjects(mrrtItem.refreshToken, @"new refresh token");
 }
 
+#pragma mark - Private
+
+- (ADAuthenticationContext *)getTestAuthenticationContext
+{
+    ADAuthenticationContext* context =
+    [[ADAuthenticationContext alloc] initWithAuthority:TEST_AUTHORITY
+                                     validateAuthority:NO
+                                                 error:nil];
+    
+    NSAssert(context, @"If this is failing for whatever reason you should probably fix it before trying to run tests.");
+    ADTokenCache *tokenCache = [ADTokenCache new];
+    [context setTokenCacheStore:tokenCache];
+    [context setCorrelationId:TEST_CORRELATION_ID];
+    
+    return context;
+}
+
+
+- (ADTestURLResponse *)defaultTokenEndpointPkeyAuthChallenge
+{
+    return [self adDefaultRefreshReponseCode:401
+                             responseHeaders:@{ @"WWW-Authenticate" : @"PKeyAuth nonce=\"AAABAAEAiL9Kn2Z27UubvWFPbm0gLdtsn-PXocm89MSCN-jy-PyMb1txkhQMWoFNUDgLkmMs1OnKIexU4jwre7oqMSKjpKk3wjvHvJlE6ZFBdeEKVQtd_IXHzbR9wT-obZUI5kM779akwJHoPQ4aBnGlrbqUTCAA\", CertAuthorities=\"OU=82dbaca4-3e81-46ca-9c73-0950c1eaca97,CN=MS-Organization-Access,DC=windows,DC=net\", Version=\"1.0\", Context=\"pkeyauth_context\"" }
```

Comment:
It is hard to say if this token is real or not. Maybe it was specifically generated for the tests. Even if it was borrowed from a real server response, it is expired now, so there is no harm to have it in test. @oldalton @RPangrle please correct me if I am wrong.